eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -0,0 +1,177 @@
+# Helper functions for retireving useful information from azure-sdk-for-* repo
+# Example Use : Import-Module .\eng\common\scripts\modules
+class PackageProps
+{
+    [string]$pkgName
+    [string]$pkgVersion
+    [string]$pkgDirectoryPath
+    [string]$pkgReadMePath
+    [string]$pkgChangeLogPath
+
+    PackageProps()
+    {
+    }
+    PackageProps(
+        [string]$pkgName,
+        [string]$pkgVersion,
+        [string]$pkgDirectoryPath
+    )
+    {
+        $this.pkgName = $pkgName
+        $this.pkgVersion = $pkgVersion
+        $this.pkgDirectoryPath = $pkgDirectoryPath
+
+        if (Test-Path (Join-Path $pkgDirectoryPath "README.md"))
+        {
```

Comment:
The bracing style is a little off typically you would see braces like one of the following:

```
       if (Test-Path (Join-Path $pkgDirectoryPath "CHANGELOG.md")) {
            $this.pkgChangeLogPath = Join-Path $pkgDirectoryPath "CHANGELOG.md"
       } 
       else {
           $this.pkgChangeLogPath = $null
       }
```
or

```
       if (Test-Path (Join-Path $pkgDirectoryPath "CHANGELOG.md")) 
       {
            $this.pkgChangeLogPath = Join-Path $pkgDirectoryPath "CHANGELOG.md"
       } 
       else
       {
           $this.pkgChangeLogPath = $null
       }
```

I generally use the first approach for PS scripts to try and shorten number of lines.